examples/ts-ganttchart-d3/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ts-ganttchart-d3",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "server": "mods-dev-server dist",
+    "build": "webpack --mode production",
+    "build-watch": "webpack --watch",
+    "start": "npm run build-watch",
+    "test": "jest --watchAll"
+  },
+  "author": "cnicolet, valexand, ebrandin",
+  "license": "ISC",
+  "dependencies": {
+    "@types/d3": "^6.7.0",
+    "d3": "^7.0.0",
+    "invert-color": "^2.0.0",
+    "moment": "^2.29.1"
```

Comment:
Will we need any legal/black duck work for these new dependencies?

Reply:
I don`t know. They have MIT license.